Refactor deployment related release notes (#1965)
ballerina-platform/ballerina-release: release-notes/swan-lake-2201.1.0-release-note.md

To view bug fixes, see the [GitHub milestone for Swan Lake 2201.1.0](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%222201.1.0%22+label%3AType%2FBug).

> ### Deployment Updates

Reviewer: ```suggestion
### Deployment updates
```

#### Improvements
- Reduced the package size of `ballerina/cloud`
- Docker image generation now relies on the user's docker client

> - The `ballerinax/awslambda package` is now available at central

Reviewer: ```suggestion
- The `ballerinax/awslambda package` is now available in [Ballerina Central](https://central.ballerina.io/ballerinax/aws.lambda)
```
Please check if the URL is correct.

> - The `ballerinax/azure_functions package` is now available at central

Reviewer: ```suggestion
- The `ballerinax/azure_functions package` is now available in [Ballerina Central](https://central.ballerina.io/ballerinax/azure.functions)
```
Please check if the URL is correct.

> #### Breaking Changes

Reviewer: ```suggestion
#### Breaking changes
```

> - For existing `ballerinax/awslambda` projects, change the version to `2.1.0` in the `Dependencies.toml` file.

Reviewer: ```suggestion
For existing `ballerinax/awslambda` and `ballerinax/azure_functions` projects, change the version to `2.1.0` in the `Dependencies.toml` file.
```

> - For existing `ballerinax/azure_functions` projects, change the version to `2.1.0` in the `Dependencies.toml` file.

Reviewer: ```suggestion

```
